README.md

```diff
@@ -8,13 +8,89 @@ This repository is the counterpart to the [Jenkins Shared Library repository](ht
 
 #### python.setup.py.yml
 
-This job is based on setuptools and runs builds, linter and tests using `python setup.py` commands.
+This job is based on setuptools and does the following:
 
-### Steps
+* builds the Python module
+* validates the module (eg. linter, tests)
+* deploys the module to a PyPI index
+* deploys the module as a Docker image to a Docker registry
 
-#### python.build.yml
+Before using the pipeline please create a new [variable group](https://docs.microsoft.com/en-us/azure/devops/pipelines/library/variable-groups?view=azure-devops&tabs=yaml)
+with the name `GitHub`. This group contains variables required during the release/deploy
+process.
 
-#### python.publish.yml
+Add the following variables to the group:
+
+* GH_TOKEN
+
+  The [personal access token](https://github.com/settings/tokens) is used when pushing the version bump commits to the master branch during a release (see details below in *Variables -> release*). Change the variable type to *secret*.
+* GH_USER_MAIL
+
+  The email of the user assigned to the personal access token (*GH_TOKEN*).
+
+  Example: `123847392+my-ci-user@users.noreply.github.com`
+* GH_USER_NAME
+
+  The name of the user assigned to the personal access token.
+
+  Example: `my-ci-user`
+
+##### Variables
+
+###### release
+
+The Python pipeline allows only on-demand releases. By default the project is only built and validated. If a new release of the Python module and Docker image
+should be released a new build has to be queued.
+
+In order to initiate a new release the variable *release: true* needs to be defined during the queuing. This will result in bumping the
+version (snapshot->release) and result in a deployed PyPI package, a Docker image and two version commits.
+
+In order
+
+###### dockerSnapshot
+
+By default Docker images are built only during the release process. In case a snapshot image is needed one can queue a new build and set the variable
+*dockerSnapshot: true*. This will result in building a Docker image with a snapshot version.
+
+##### Parameters
+
+###### dockerDeploy
+
+Flag to enable/disable building and publishing a Docker image with the Python module.
+
+*Default*: true
+
+###### dockerDeployFile
+
+Filename of the Dockerfile used for defining the content of the image to be pushed to the Docker registry.
+
+*Default*: Dockerfile
+
+###### dockerRegistryConnector
+
+Azure DevOps service connector that defines the Docker registry where the Docker image should be pushed to.
+
+*Default*: tomtom-docker-registry-bintray
+
+###### dockerRepo
+
+Name of the repository an image should be pushed to. This is as well called in Docker terminology a namespace.
```

Comment:
“Also called a namespace in Docker terminology”

Reply:
Fixed